andela/prometheus-ah
# #159987738 Implemented the articles crud endpoint

**What does this do**
Have CRUD endpoint on articles

**Description of the task completed**
Have the following endpoints working
`GET api/articles/`
`GET api/articles/<article_id>`
`POST api/articles/`
`PUT api/articles/<article_id>`
`DELETE api/articles/<article_id>`

**How should this be manually tested**
- Clone repo
- Run `npm install`
- create a `.env` file using the sample and supply appropriate values
- Run `npm run test`
- Run in postman

**Note:** Authorization is needed to access the `create update delete` endpoints. Authorization should be added to the header
Authorization value: JWT

**Relevant Pivotal Tracker Story**
- https://www.pivotaltracker.com/story/show/159987738
